### DISCUSSION TITLE
How to convert pngs to tif with layers?

### DISCUSSION THREAD

**Discussion by WaterYep** [ImageMagick/ImageMagick#5194] [Help] [answered]
ImageMagick version
7.1.0-36

Operating system
Linux

Operating system, version and so on
Ubuntu  20.04

Description
When I use 
            'convert\
            \( -page +0+0 ./img1.png   \)\
            \( -page +0+0 ./img2.png   \)\
            \( -clone 0--1 -background none -mosaic \) -alpha On\
            -reverse -compress Lossless ./result.tif'
It just have one layer

**Comment by snibgo** [comment, marked as answer]
It works fine for me, on IM v7.1.0-32.

I suggest using "magick" instead of "convert", though that shouldn't make any difference.

Please show the text output of "magick ./result.tif info:"

**Reply by WaterYep** [reply]
I misremembered the imagemagick version，it works well , Thanks!
